src/bundle/Resources/views/parts/menu/top_menu_2nd_level.html.twig

```diff
@@ -27,7 +27,7 @@
     {% set linkAttributes = item.linkAttributes|merge({
         'href': item.uri,
         'aria-controls': item.name,
-        'class': 'nav-link' ~ (matcher.isCurrent(item) ? ' ' ~ options.currentClass)
+        'class': ('nav-link' ~ (matcher.isCurrent(item) ? ' ' ~ options.currentClass|trim)) ~ (item.linkAttributes.class is defined ?  ' ' ~ item.linkAttributes.class|trim)
```

Comment:
As above:
`'class': ('nav-link' ~ (matcher.isCurrent(item) ? ' ' ~ options.currentClass|trim)) ~ ' ' ~ item.linkAttributes.class|default(''))|trim`

Haven't tested it though.